tests/cli/dorq/ui/test_presenters.py

```diff
@@ -350,7 +350,7 @@ def test_open_url_in_browser(self, success: bool, monkeypatch: pytest.MonkeyPatc
     26,
     7,
     704015,
-    tzinfo=timezone(timedelta(hours=-5)),
+    tzinfo=timezone.utc,
```

Comment:
**non-blocking**: is this change still needed after mocking out the formatter function?